pkg/api/meta/meta_test.go

```diff
@@ -739,6 +739,8 @@ type MyAPIObject2 struct {
 func getObjectMetaAndOwnerRefereneces() (myAPIObject2 MyAPIObject2, metaOwnerReferences []metatypes.OwnerReference) {
```

Comment:
@lavalamp this commit addresses your last comments and contains some trivial changes that you haven't reviewed in your last pass.